docs/modules/ROOT/nav.adoc

```diff
@@ -45,6 +45,7 @@
 ** xref:configuration/runtime-config.adoc[Runtime configuration]
 ** xref:configuration/runtime-resources.adoc[Runtime resources]
 ** xref:configuration/runtime-version.adoc[Camel version]
+** xref:configuration/env-vars.adoc[Environment Variables]
```

Comment:
I think this one belongs more to installation>advanced and we can name it `Operator Tuning`

Reply:
Even better. Don't add a new page. Put this content instead as a paragraph in the operator tuning page: https://camel.apache.org/camel-k/next/installation/advanced/advanced.html